chart/values.schema.json

```diff
@@ -1066,6 +1076,44 @@
                     }
                 }
             }
+        },
+        "kerberos": {
+            "description": "Kerberos configurations for airflow",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Enable kerberos.",
+                    "type": "boolean"
+                },
+                "ccacheMountPath": {
+                    "description": "Path to mount shared volume for kerberos credentials cache.",
+                    "type": "string"
+                },
+                "ccacheFileName": {
```

Comment:
```suggestion
                "cacheFileName": {
```

Reply:
"ccache" stands for "credential cache" and is pretty common in the Kerberos world. https://web.mit.edu/kerberos/krb5-1.12/doc/basic/ccache_def.html

Reply:
aah, TIL :)